.github/workflows/ci.yml

```diff
@@ -93,6 +100,7 @@ jobs:
         run: cargo install cargo-nextest --locked
 
       - name: Run tests
+        if: ${{ matrix.os == 'windows-latest' }}
```

Comment:
Why restricting the full tests only on `windows`?

Reply:
Really, why...